scene/2d/collision_object_2d.cpp

```diff
@@ -29,6 +29,7 @@
 /**************************************************************************/
 
 #include "collision_object_2d.h"
+#include "path_2d.h"
```

Comment:
Except for the header associated with this source file all includes are absolute, should be below as: `scene/2d/path_2d.h`